autopush/config.py

```diff
@@ -283,28 +283,29 @@ def from_argparse(cls, ns, **kwargs):
                                 "Invalid client_certs argument")
                         client_certs[sig] = name
 
-        if ns.fcm_enabled:
-            fcm_core = {
-                "ttl": ns.fcm_ttl,
-                "dryrun": ns.fcm_dryrun,
-                "max_data": ns.max_data,
-                "collapsekey": ns.fcm_collapsekey}
-            if len(ns.fcm_auth) > 0:
-                if not ns.fcm_senderid:
-                    raise InvalidConfig("No SenderID found for FCM")
-                fcm_core.update({
-                    "version": 0,
-                    "auth": ns.fcm_auth,
-                    "senderID": ns.fcm_senderid})
-            if len(ns.fcm_service_cred_path) > 0:
-                fcm_core.update({
+        if ns.fcm_enabled and ns.fcm_creds:
```

Comment:
`fcm_enabled` is now deprecated though?

Reply:
thanks, forgot to get that. (TIL `parser.add_argument` still sets boolean state if the arg exists and no other options set.)